sympy/combinatorics/tests/test_schur_number.py

```diff
@@ -0,0 +1,29 @@
+from sympy.combinatorics.schur_number import schur_partition
+import random
+
+def _sum_free_test(subset):
+    """
+    Checks if subset is sum-free(There are no x,y,z in the subset such that
+    x + y = z)
+    """
+    for i in subset:
+        for j in subset:
+            if i + j in subset:
+                raise AssertionError("This subset is not sum free")
+
+def test_schur_number():
+    n = random.randint(1,1000)
```

Comment:
What about setting the seed for deterministic tests?

Reply:
@czgdp1807 I was planning to do that soon. Adding some deterministic and some random tests.

Reply:
IMO, adding non-deterministic tests would put the master build at risk. I would say set the set if it isn't already set for any test using random.

Reply:
Ok @czgdp1807 I will set the seed values and probably run these random tests 4 to 5 times.